pkg/boot/universalpayload/utilities_arch_amd64.go

```diff
@@ -65,7 +65,7 @@ func getPhysicalAddressSizes() (uint8, error) {
 // Due to lack of support to set value of General Purpose Registers in kexec,
 // bootloader parameter needs to be prepared in trampoline code.
 // Also stack is prepared in trampoline code snippet to ensure no data leak.
-func constructTrampoline(buf []uint8, hobAddr uint64, entry uint64) []uint8 {
+func constructTrampoline(buf []uint8, addr uint64, entry uint64) []uint8 {
```

Comment:
isn't hobAddr more descriptive than just `addr`?

Reply:
the layout changed. Now we can use the base address of 'available memory region' as the loaded address, and use component offset to calculate base address of each components.